app/css/components/site-header.css

```diff
@@ -18,6 +18,11 @@ body:not(.namespace-.controller-insiders) {
     --exercism-link-filter: var(--textColor1-filter);
 }
 
+body.keyboard-navigation .nav-element:hover .nav-element-dropdown,
+body.keyboard-navigation .nav-element:hover .arrow {
+    display: none !important;
```

Comment:
I believe you can do `@apply !hide`; or something if we want to use Tailwind here,